script/release.sh

```diff
@@ -134,6 +135,13 @@ git_pull() {
   cd ..
 }
 
+
+git_sync() {
+  cd $CLI
+  git pull upstream
```

Comment:
I still think this won't work. You should also specify branch.
`git pull upstream master` I would probably use fetch here. `git fetch upstream master`